.github/workflows/main.yml

```
@@ -0,0 +1,61 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ${{ matrix.os }}
+    container: ${{ matrix.container }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
```

Why not include Windows?
```suggestion
        os: [ubuntu-latest, windows-latest]
```